scripts/helpers.py

```diff
@@ -136,7 +136,10 @@ def format_dt_monitors_yaml(department, data_filtered, environment):
             "locations": ["SYNTHETIC_LOCATION-CC3E4D2657A13D18"],
             "requests": [
                 {
-                    "url": f'http://{item["host"]}/health',
+                    if "internal" in {item["host"]}:
```

Comment:
```suggestion
                    if item["host"].endswith("internal"):
```
Perhaps this would be a better option?

Reply:
Sure